robotoff/app/core.py

```diff
@@ -111,7 +111,8 @@ def get_insights(
 
     if order_by is not None:
         if order_by == "random":
-            query = query.order_by((peewee.fn.Random() * ProductInsight.n_votes).desc())
+            # The +1 is here to avoid 0*rand() = 0
+            query = query.order_by((peewee.fn.Random() * (ProductInsight.n_votes + 1)).desc())
```

Comment:
:clap: really good catch !

Reply:
BTW it made me think of adding #664 bug about performances problem of this piece of code.